components/server/src/auth/bearer-authenticator.ts

```diff
@@ -103,4 +136,26 @@ export class BearerAuth {
 
         req.user = user;
     }
+
+    private parsePersonalAccessToken(token: string): { prefix: string; signature: string; value: string } {
```

Comment:
Would it make sense to have at least a rudimentary test for this?

Reply:
added.